include/nbl/video/IPhysicalDevice.h

```diff
@@ -143,6 +143,8 @@ class IPhysicalDevice : public core::IReferenceCounted
         return false;
     }
 
+    virtual ~IPhysicalDevice() = 0 {}
+
```

Comment:
wtf?

Reply:
you don't want a `virtual` destructor for `IPhysicalDevice` for some reason?

Reply:
okay my bad, looks like deriving from `Interface` provides you with a `virtual` destructor

Reply:
resolved